Add docker image (#14)
robgietema/nick: README.md

    $ yarn bootstrap
    $ yarn start

## Using Docker Image

Make sure you have docker installed in your system.

### Backend

> Run `docker compose up --build` in the root directory to run the docker container for backend.

Reviewer: ````suggestion
Navigate to the root of the repository, and run the following command to run the Docker container for backend.

```shell
docker compose up --build
```
````

> This will expose port '8000' and you can make requests to 'http://localhost:8000/++api++/' to fetch content.

Reviewer: ```suggestion

This will expose port `8000`.
You can make requests to `http://localhost:8000/++api++/` to fetch content.
```

> Run `docker compose down` to shut down the containers.

Reviewer: ````suggestion
To shut down the containers, run the following command.

```shell
docker compose down
```
````

### Frontend

> Use any frontend as you feel like. For instance, you can go to `/frontend` directory and use pre-built frontend.

Reviewer: ```suggestion
You can use any frontend.
For convenience, you can navigate to the directory `/frontend` and use the pre-built frontend.
```
